playwright-typescript-playwright-test-main/playwright-typescript-playwright-test-main/tests/functional/Login.test.ts

```diff
@@ -1,7 +1,22 @@
-import test from '@lib/BaseTest';
 
-test(`Login to HRMS`, async ({ loginPage }) => {
-    await loginPage.navigateToURL();
-    await loginPage.loginToDotnetEmployee();
+import {test} from '@playwright/test'
+import LoginPage  from '@pages/LoginPage'
+import { LoginPageObjects } from '@objects/LoginPageObjects';
+
+test.describe('Login Page', async () => {
+    let loginpageobjects:LoginPageObjects;
+    let loginpage:LoginPage;
+
+test.beforeEach(async ({page})=>{
```

Comment:
this Test is bout logging into the application with valid credentials.. then why we have beforeEach